### DISCUSSION TITLE
CVE-2024-7646

### DISCUSSION THREAD

**Discussion by ultramaxim** [nginx/kubernetes-ingress#6304] [General]
Hello! are you affected to CVE-2024-7646 ?

**Comment by AlexFenlon** [comment]
Hi @ultramaxim,

From what we understand this CVE affected the Community Ingress Controller.
We are currently confirming if we are affected or not.

**Comment by haywoodsh** [comment]
The community Ingress Controller and NGINX Ingress Controller are different projects and the implementations are complete different. We don't perform Regex validation the same way.
